### PR TITLE
add changes to teams form feature #40

### PR DESCRIPTION
**Pull request by fernandoamz**

### CODE REVIEW

File: app/assets/stylesheets/scaffolds.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px;
+  font-family: verdana, arial, helvetica, sans-serif;
+  font-size: 13px;
+  line-height: 18px;
```
Comment (diegosaul402) [inline app/assets/stylesheets/scaffolds.scss]: Please sort these properties alphabetically, on every element.

File: app/views/teams/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+<h1>Teams</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Principal position</th>
+      <th>Start date</th>
+      <th>End date</th>
```
Comment (diegosaul402) [inline app/views/teams/index.html.erb]: Please localize those strings.

File: app/assets/stylesheets/scaffolds.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
+body {
+  background-color: #fff;
```
Comment (8geonirt) [inline app/assets/stylesheets/scaffolds.scss]: Remember to use variables to store those hexadecimal values.

File: app/assets/stylesheets/scaffolds.scss [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+pre {
+  background-color: #eee;
+  font-size: 11px;
+  padding: 10px;
+}
+
+a {
+  color: #000;
```
Comment (8geonirt) [inline app/assets/stylesheets/scaffolds.scss]: Same thing about colors.

File: app/models/team.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+class Team < ApplicationRecord
```
Comment (8geonirt) [inline app/models/team.rb]: Is this model not going to have any validations?

File: config/locales/en.yml [hunk 6]
```diff
@@ -25,4 +25,23 @@ en:
     information: "Lorem ipsum..."
   
   banner:
-    title: "Finden"
\ No newline at end of file
+    title: "Finden"
+
+  team: 
+    alert_message: "¿Are you sure?"
```
Comment (8geonirt) [inline config/locales/en.yml]: Typo: remove the `¿`, in English' questions that character doesn't exist.

File: config/locales/es.yml [hunk 7]
```diff
@@ -25,4 +25,23 @@ es:
     information: "Lorem ipsum..."
 
   banner:
-    title: "Finden"
\ No newline at end of file
+    title: "Finden"
+
+  team: 
+    alert_message: "¿Estás seguro?"
+    back: 'Atras'
```
Comment (8geonirt) [inline config/locales/es.yml]: Typo: it should be _Atrás_

File: test/controllers/teams_controller_test.rb [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+  end
+
+  test "should get new" do
+    get new_team_url
+    assert_response :success
+  end
+
+  test "should create team" do
+    assert_difference('Team.count') do
+      post teams_url, params: { team: { end_date: @team.end_date, name: @team.name, principal_position: @team.principal_position, start_date: @team.start_date } }
```
Comment (8geonirt) [inline test/controllers/teams_controller_test.rb]: Let's split this line to make it more readable:
Something like:
```
post teams_url, params: {
    team: {
      end_date: @team.end_date,
      name: @team.name,
      principal_position: @team.principal_position,
      start_date: @team.start_date
    }
  }
```

File: test/controllers/teams_controller_test.rb [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get edit_team_url(@team)
+    assert_response :success
+  end
+
+  test "should update team" do
+    patch team_url(@team), params: { team: { end_date: @team.end_date, name: @team.name, principal_position: @team.principal_position, start_date: @team.start_date } }
```
Comment (8geonirt) [inline test/controllers/teams_controller_test.rb]: Same here, split those lines 🤓

**Pull request merged**